packages/payment-detection/src/provider.ts

```diff
@@ -0,0 +1,21 @@
+import { ethers } from 'ethers';
```

Comment:
important. This is a workaround for the default infura key, and it will allow to customize the ApiKey if needed